index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Medical core</title>
+</head>
+<body>
+    <div class="medical-header__container">
+        <header class="medical-header">
+            <h1 class="medical-header__logo"> <a class="medical-header__anchor" href="">Medical Core</a></h1>
+            <ul class="medical-header__list">
+               <li class="medical-header__item"><a class="medical-header__anchor" href="">Home</a></li>
+               <li class="medical-header__item"><a class="medical-header__anchor" href="">Product</a></li>
+               <li class="medical-header__item"><a class="medical-header__anchor" href="">Pricing</a></li>
+               <li class="medical-header__item"><a class="medical-header__anchor" href="">Contact</a></li>
+            </ul>
+    
+            <ul class="medical-header__list">
+                <li class="medical-header__item--login"><a class="medical-header__anchor--login" href="">Login</a></li>
+                <li class="medical-header__item--signup"><a class="medical-header__anchor--signup" href="">Join Us</a></li>
+            </ul>
+        </header>
+
+
+        <div class="medical-banner">
+            <h4 class="medical-banner__header">Welcome</h4>
+            <p class="medical-banner__paragraph--1">A Great Place to Receive Care </p>
+            <p class="medical-banner__paragraph--2">Get your best looking smile now!</p>
+    
+            <ul class="medical-banner__list">
+                <li class="medical-banner__list__item medical-banner__list__item--join"> Join Us</li>
+                <li class="medical-banner__list__item medical-banner__list__item--learn">Learn More</li>
+            </ul>
+        </div>
+
+    </div>
+  
+    <div class="medical-expert">
+        <div class="medical-expert__img-wrapper">
+          <img
+            src="./assets/main-image.png"
+            alt="expert"
+            class="medical-expert__img"
+          />
+        </div>
+        <div class="medical-expert__body">
+          <div class="medical-expert__body-border"></div>
+          <h2 class="medical-expert__title">Meet Our Experts</h2>
+          <p class="medical-expert__text">
+            Problems trying to resolve the conflict between the two major realms
+            of Classical physics: Newtonian mechanics
+          </p>
+          <a href="#" class="medical-expert__button"
+            >Learn More </a>
+        </div>
+    </div>
+
+    <div class="team-container">
+        <h5 class="team-title">Team</h5>
+        <p class="team-subtitle">Our Team</p>
+        <p class="team-paragraph">Problems trying to resolve the conflict between 
```

Comment:
here you are making `team-title` `team-subtitle` `team-paragraph` you put them as a block but I don't see where you are using them